.evergreen/sign-packaged-artifact.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+set -e
+set -x
+
+export NODE_JS_VERSION=${NODE_JS_VERSION}
+export ARTIFACT_URL_FILE="$PWD/../artifact-url.txt"
+
+# TODO: Remove this once it is confirmed that it's safge to do so
```

Comment:
do we need to link the ticket here?

Reply:
sorry, but do we even need this as we are using garasign? (or is this particular build a special one?)

Reply:
Yeah, thank you for the reminder! I think we can just remove this, that's exactly what I was hoping for, just forgot about this TODO myself 😅